test/parallel/test-timers-ordering.js

```diff
@@ -25,7 +25,7 @@
 require('../common');
 const assert = require('assert');
 const { internalBinding } = require('internal/test/binding');
-const { getLibuvNow } = internalBinding('timers');
+const binding = internalBinding('timers');
```

Comment:
Just curious... why change this?

Reply:
The fast API calls need to get the binding data from somewhere, and the best place would be the receiver (it would not hit the fast path anyway if the receiver isn't an actual object).

Reply:
Oy that makes sense but it's a bit of a footgun. Definitely something worth documenting here

Reply:
Having a lint rule for this would be a good idea, if we stick with this pattern.

Reply:
Added some comments in the call sites for this. I suppose linting might help (though even in this case only a subset of the bindings need to be called with it), but that should be done separately